### PR TITLE
feat: vanilla ae with independent channels 

### PR DESCRIPTION
**Pull request by ab93**
Note: Will remove data

### CODE REVIEW

File: numalogic/models/autoencoder/variants/icvanilla.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        n_features: num of features
+        layersizes: encoder layer size
+        dropout_p: the dropout value
+
+    """
+
+    def __init__(
+        self,
+        seq_len: int,
+        n_features: int,
```
Comment (mboussarov) [inline numalogic/models/autoencoder/variants/icvanilla.py:26]: Have to decide if to go with n_features or n_channels name and have it consistent with the MultichannelAE

**Review by mboussarov — APPROVED** [review]
Looks good!

File: numalogic/models/autoencoder/variants/icvanilla.py (hunk 1 above)
Comment (ab93) [reply numalogic/models/autoencoder/variants/icvanilla.py:26]: Changed.

File: numalogic/config/factory.py [hunk 2]
```diff
@@ -146,6 +147,7 @@ class ModelFactory(_ObjectFactory):
         "TransformerAE": TransformerAE,
         "SparseTransformerAE": SparseTransformerAE,
         "Conv1dVAE": Conv1dVAE,
+        "VanillaICAE": VanillaICAE,
```
Comment (s0nicboOm) [inline numalogic/config/factory.py:150]: This name looks little odd.

**Review by s0nicboOm — APPROVED** [review]
knitpick otherwise lgtm

File: numalogic/config/factory.py (hunk 2 above)
Comment (ab93) [reply numalogic/config/factory.py:150]: Will change later once Miro verifies the usage

**Pull request merged**